biodata-tools/pom.xml

```diff
@@ -91,6 +132,12 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <!--TODO Review -->
+        <dependency>
```

Comment:
Pretty sure this one is not needed...
https://github.com/opencb/biodata/commit/b839ae16e79ee1d31f4b1583920001be99073c84